v1.0/frequently-asked-questions.md

```diff
@@ -125,6 +125,12 @@ CockroachDB supports the PostgreSQL wire protocol, so you can use any available
 
 See [Install Client Drivers](install-client-drivers.html) for more details.
 
+## Why does CockroachDB use the PostgreSQL wire protocol instead of the MySQL protocol?
+
+CockroachDB uses the PostgreSQL wire protocol because it is better documented than the MySQL protocol and the PostgreSQL license is better for us than the MySQL license.
```

Comment:
PostgreSQL uses the PostgreSQL license: "a liberal Open Source license, similar to the BSD or MIT licenses."

MySQL is licensed under the GPL which is much more restrictive. @bdarnell can provide better words here.